### ISSUE TITLE
Suggested actions are not spoken

### ISSUE THREAD

**Issue by leolorenzoluis** [microsoft/BotFramework-WebChat#1780]
Given the server returns a reply that has SuggestedActions with Speak property populated. The WebChat wouldn't speak the given reply by the server.

**Comment by corinagum** [comment]
According to the [documentation](https://docs.microsoft.com/en-us/azure/bot-service/rest-api/bot-framework-rest-connector-api-reference?view=azure-bot-service-4.0#cardaction-object), there is no Speak property on the Suggested Actions schema. 

Could you recommend scenarios where the Speak property is necessary? If so, this will help us  to plan for this feature.

**Comment by leolorenzoluis** [comment]
Accessibility. What if the person can't see and only interacts based on what he hears?

**Comment by corinagum** [comment]
thank you! I think I closed this on accident last week so I'm going to keep it open for discussion with the dev team.

**Comment by corinagum** [comment]
We need to add aria-live to the suggested-actions buttons. We'll fix this targeting for 4.5

**Comment by leolorenzoluis** [comment]
Will adding the aria tag make the bot speak without using accessibility tools?

**Comment by compulim** [comment]
No, `aria-live` tag only works with changing content. But not initial content.

Today, we add a box (element) around suggested actions when we need to show them up. We remove the element when there are no suggested actions.

We cannot simply add `aria-live` tag in that box, because it is not "changing content", it is "creating a new element". And `aria-live` will not speak out when the element is new.

So, we need to have the box persisted on Web Chat UI, and change its content from time to time.

On the other side, while we are experimenting with `aria-live` region, we found that a page with multiple `aria-live` regions may conflict with each other. So if it doesn't work (multiple `aria-live` regions in a our setup), then, we might need a bigger change on DOM structure to make it work.

**Comment by compulim** [comment]
More clarifications over my previous comment.

I think `role="status"` is conflicting with `aria-live="polite"`. But as @corinagum pointed out, two `aria-live="polite"` should not conflict with each others.

**Comment by leolorenzoluis** [comment]
Just following up, does this mean the app will need a narrator in order for the person to hear what the bot says? It still won't speak the first time it starts a conversation?

**Comment by corinagum** [comment]
This issue is in regards to the accessibility of Suggested Actions. Yes, customers will need an Assistive Technology to be alerted of a change in Suggested Actions. This is not related to speech. As discussed above, Suggested Actions to not have a speak property and therefore will not be spoken by the (speech) bot by design.
